content/en/contribute/code/releasing/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Releasing"
+linkTitle: "Releasing"
+weight: 7
+description: >
+  Instructions for releasing CHT tools
+aliases: >
+  /core/guides/releasing
+---
+
+## CHT Core
+
+A release is a set of code changes bundled together, ideally with at least one deployment of CHT apps ready to make use of it.
+
+### Building & Releasing CHT Core Changes
+
+The high-level steps for a release are as follows:
+
+* The [Focused Working Group]({{% ref "/contribute/medic/product-development-process/focused-groups" %}}) sees an opportunity they want to go after. The opportunity addresses a need of at least one CHT app deployment and will be used by that deployment after the release.
+* The Focused Working Group agrees on a solution for it.
+* Tickets are added to the [Product Team Activities board](https://github.com/orgs/medic/projects/134/views/3) for what's being built.
+* A [release manager](#release-manager) is assigned from the team.
+* The release manager [creates an issue](https://github.com/medic/cht-core/issues/new/choose) for either a [Major/Minor or Patch](#majorminorpatch-release) release issue template and follows the process.
+* Code is built by a developer together with [quality assistance]({{% ref "/contribute/medic/product-development-process/quality-assistance" %}}).
+* [Code is reviewed]({{% ref "/contribute/code/workflow#code-reviews" %}}).
+* Code is merged.
+* Code is released.
```

Comment:
Hey @andrablaj , if this documentation is mainly meant for external use, is it possible to add a few words that explain what 'code is released' mean e.g, it is made available to the public.

Reply:
Thank you, @esthermmoturi, for the feedback! We already explained at the beginning of the page that:

> A release is a set of code changes bundled together, ideally with at least one deployment of CHT apps ready to make use of it.

I believe that should be sufficient for the readers to understand the page's purpose.